Locales/ptBR.lua

```diff
@@ -4,32 +4,32 @@ local _, mod = ...
 local L = mod.L
 
 -- Core
-L.invasionsAvailable = "|T236292:15:15:0:0:64:64:4:60:4:60|t New invasions available!"
-L.tooltipClick = "|cffeda55fClick|r to drag and move."
-L.tooltipClickOptions = "|cffeda55fRight-Click|r to open options."
-L.searching = "Searching..."
+L.invasionsAvailable = "|T236292:15:15:0:0:64:64:4:60:4:60|t Novas invasões disponíveis!"
+L.tooltipClick = "|cffeda55fClick|r para segurar e mover."
```

Comment:
Missing translation for `Click` and `Right-Click`
Still missing `LegionInvasionTimer.toc`